crates/matrix-sdk-sqlite/src/lib.rs

```diff
@@ -35,3 +38,53 @@ pub use self::state_store::SqliteStateStore;
 
 #[cfg(test)]
 matrix_sdk_test::init_tracing_for_tests!();
+
+/// A configuration structure used for opening a store.
+pub struct StoreOpenConfig {
```

Comment:
Perhaps a better name would be `StoreConfig` or `StoreConfiguration`. Maybe even mention SQLite somewhere since this is specific for the SQLite store implementation.

Reply:
Okay to me to rename to `StoreConfig` (I don't like `Config`, I would prefer `Configuration`, but `Config` is used everywhere else in the code, consistency matters most). However, this is the `matrix-sdk-sqlite`, I wonder why we need to prefix everything by `Sqlite`, but again… consistency matters most. I'm changing to `SqliteStoreConfig`.

Reply:
>  I wonder why we need to prefix everything by Sqlite

People complain about too many types with the same name. I think the complaint is warranted.